test/suite/functional/Generator/Request/GetResourcesRequest.php

```diff
@@ -21,6 +21,17 @@ class GetResourcesRequest implements RequestInterface
     /** @var int[]|null */
     private $filterByIds;
 
+    /** @var string */
+    private $contentType = '';
+
+    /**
+     * @return string
```

Comment:
cs fixer should remove superfluous phpdoc 🤔 that was part of the latest commit I've done, maybe it's not rebased with master?